nrpy/equations/nrpyelliptic/ConformallyFlat_RHSs.py

```diff
@@ -0,0 +1,161 @@
+"""
+Construct symbolic expressions for the right-hand-side of the hyperbolic relaxation equation.
+Curvilinear coordinates are supported using a reference-metric formalism.
+
+Authors: Thiago Assumpção; assumpcaothiago **at** gmail **dot* com
+         Zachariah B. Etienne; zachetie **at** gmail **dot* com
+
+License: BSD 2-Clause
+"""
+
+# Step P1: Import needed modules:
+import sympy as sp  # For symbolic computations
+import nrpy.grid as gri  # NRPy+: Functionality for handling numerical grids
+import nrpy.indexedexp as ixp  # NRPy+: Symbolic indexed expression (e.g., tensors, vectors, etc.) support
+import nrpy.reference_metric as refmetric  # NRPy+: Reference metric support
+from nrpy.equations.nrpyelliptic.CommonParams import eta_damping
+
+
+# Specify RHSs as class variables,
+# to enable access outside this
+# function (e.g., for C code output)
+class RelaxationEquationCurvilinear_RHSs:
```

Comment:
Maybe HyperbolicRelaxationRHSGenerator would be a better name for this class?

Reply:
or even HyperbolicRelaxationCurvilinearRHSs

Reply:
I changed the class name to `HyperbolicRelaxationCurvilinearRHSs` here https://github.com/nrpy/nrpy/pull/17/commits/cc8ed44eab31459fb4cf5df7b9e7c714f44b133d.